MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: Manually create DataFormItems in Maui DataForm control | Syncfusion
+description: Learn here all about Working with manually created DataFormItem's in Syncfusion .NET MAUI DataForm(SfDataForm) control.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+## Adding DataFormItems Manually
+
+Support has been provided to generate DataFormItems manually for the defined business model using the [Items]`Link` property of the `SfDataForm` class. You need to set the [AutoGenerateItems]`Link` property to false to restrict the auto-generation of DataFormItems. 
```

Comment:
You can manually create [DataFormViewItem] for the defined business model by using the [Items] property of the 'SfDataForm' class. You should set the [AutoGenerateItems] property to false o restrict the auto-generation of [SfDataForm.Items].

Reply:
above mentioned content updated